package.json

```diff
@@ -38,6 +38,7 @@
     "mem": "^9.0.2",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
+    "react-scripts": "5.0.1",
```

Comment:
How was this project running without `react-scripts` before 😄 I can see it is a CRA app. CRA app should only be used with small practice projects, so we might need to discuss if we can move/refactor this project away from CRA. Either eject and start using Vite or move entirely to NextJS. cc @urandu

Reply:
@akhenda we shall rely on your guidance on this. To get things off the ground we went for the simplest approach but knowing that along the way, we'll get more experienced contributors to steer the project.